Frechet_distance/doc/Frechet_distance/Concepts/FrechetDistanceTraits.h

```diff
@@ -34,5 +38,50 @@ using Kernel = unspecified;
 */
 using Point = unspecified_type;
 
+/*! The number type of the filtered kernel. If
```

Comment:
`If` ? something missing?

Reply:
Better look at it once I ask for review. Sorry for these WIP commits.